web/src/invite.ts

```diff
@@ -593,6 +597,29 @@ function open_invite_user_modal(e: JQuery.ClickEvent<Document, undefined>): void
     });
 }
 
+export function show_email_configure_banner(): void {
+    const $email_configure_banner_container = $(".invite-banner-container");
+    if ($email_configure_banner_container.length > 0) {
+        const EMAIL_CONFIGURE_BANNER: Banner = {
+            intent: "warning",
+            label: $t({defaultMessage: "Add your email to access this feature."}),
+            buttons: [
+                {
+                    attention: "primary",
+                    label: $t({defaultMessage: "Add"}),
+                    custom_classes: "configure-email",
```

Comment:
Hmm... there's a "configure-email-visibility" class and concept, so maybe a better class name here would include "demo-organization" .... `demo-organization-add-email`?